### PR TITLE
Delete redundant commands & Clean up the codebase

### LINKED ISSUE #129: Remove redundant commands of Contacts

### LINKED ISSUE #138: Refactor all common messages of commands to Messages.java

### PR DESCRIPTION
**Pull request by UncleGrandpa925**
closes #129 & closes #138

### CODE REVIEW

**Comment by UncleGrandpa925** [comment]
@simonteozw Hi Simon

So basically in this PR, I did the following things:
- For Contact: Removed List & Clear commands
- For Lesson: Removed List & View commands
- For Module: Removed List & View commands
- For Task: Removed View command
- And I refactored all the common messages between commands to Messages.java + Standardised their names

Thank you for reviewing it!

File: src/main/java/trackitnus/commons/core/Messages.java [hunk 1]
```diff
@@ -19,4 +19,9 @@
     public static final String MESSAGE_MODULE_DOES_NOT_EXIST = "The module specified in the command doesn't exist.";
     public static final String MESSAGE_TASK_DOES_NOT_EXIST = "There is no such task in the app";
 
+    public static final String MESSAGE_DUPLICATE_CONTACT = "This contact already exists in the app";
+    public static final String MESSAGE_DUPLICATE_LESSON = "This lesson already exists";
+    public static final String MESSAGE_NOT_EDITED = "At least one field to edit must be provided.";
+    public static final String MESSAGE_DUPLICATE_MODULE = "This module already exists";
+    public static final String MESSAGE_DUPLICATE_TASK = "This task already exists in the app";
```
Comment (simonteozw) [inline src/main/java/trackitnus/commons/core/Messages.java]: I think for the messages about duplicates, all should end with `already exists`, no need `already exists in the app`.

**Review by simonteozw — COMMENTED** [review]
Looks good! just 1 comment

File: src/main/java/trackitnus/commons/core/Messages.java (hunk 1 above)
Comment (UncleGrandpa925) [reply src/main/java/trackitnus/commons/core/Messages.java]: Fixed!

**Review by simonteozw — APPROVED** [review]
LGTM

**Pull request merged**